packages/http-client-python/emitter/src/utils.ts

```diff
@@ -242,31 +242,45 @@ export function capitalize(name: string): string {
   return name[0].toUpperCase() + name.slice(1);
 }
 
+const LIB_NAMESPACE = [
+  "azure.core",
+  "azure.resourcemanager",
+  "azure.clientgenerator.core",
+  "typespec.rest",
+  "typespec.http",
+  "typespec.versioning",
+];
+
+export function getRootNamespace(context: PythonSdkContext<SdkServiceOperation>): string {
+  let rootNamespace = "";
+  if (context.sdkPackage.clients.length > 0) {
+    rootNamespace = context.sdkPackage.clients[0].namespace;
+  } else if (context.sdkPackage.models.length > 0) {
+    const result = context.sdkPackage.models
+      .map((model) => model.namespace)
+      .filter((namespace) => !LIB_NAMESPACE.includes(namespace));
```

Comment:
you can use `$.isUserDefined` like [here](https://github.com/Azure/typespec-azure/blob/main/packages/typespec-client-generator-core/src/internal-utils.ts#L591)

Reply:
I tried then find that "namespace" here is string instead of "Type" needed by `$.isUserDefined`...